docs/UsingOurScripts.md

```diff
@@ -4,14 +4,16 @@ Adopt have setup their build scripts so that you can plug in configuration files
 
 ## defaults.json
 
-This file contains the default constants and paths used in the build scripts for whichever repository it is located in. As an example, Adopt's `defaults.json` file is located [here](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/pipelines/defaults.json). If you're unsure of any of the fields, see Adopt's example map below:
+This file contains the default constants and paths used in the build scripts for whichever repository it is located in. As an example, Adopt's `defaults.json` file is located [here](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/pipelines/defaults.json). If you're unsure of any of the fields, see Adopt's example map below:
 
 ```json
 {
     // Git repository details
     "repository"             : {
-        // Git Url of the current repository.
+        // Git Url of the openjdk-build repository.
         "url"                : "https://github.com/AdoptOpenJDK/openjdk-build.git",
+        // Git Url of the current repository.
+        "pipeline_url"       : "https://github.com/AdoptOpenJDK/ci-jenkins-pipelines.git", 
         // Git branch you wish to use when running the scripts
         "branch"             : "master"
```

Comment:
Do you use the same branch for both?

Reply:
Correct, to clarify I that change was made in a previous PR - I just updated the docs. If we need to split the branch into two then we can always do that at a later date.

Reply:
In case someone else is as fussy as me, could you add a bit to the comment explaining that the branch string is used for both?